renderdoc/core/remote_server.cpp

```diff
@@ -983,6 +983,10 @@ void RenderDoc::BecomeRemoteServer(const rdcstr &listenhost, uint16_t port,
 
   if(listenRanges.empty())
   {
+#if defined(RENDERDOC_PLATFORM_GGP)
```

Comment:
I'm not a fan of leaking platform #ifdefs into general code like this. Can you explain why the whitelist isn't needed or is a problem on GGP? IIRC with the way it works it would be accessed by effectively localhost or a local IP.

If this is really needed we could add something like `Keyboard::PlatformHasKeyInput()`. Maybe `Network::DisableSecurity()` ?

Reply:
Ah, I think I misread this before, I was thinking it removes an IP list to allow list, but it actually disables it. Removed now.